### PR TITLE
Sovrn: Adapter to accept ad unit codes

### PR DESCRIPTION
**Pull request by ikagotso**

### CODE REVIEW

File: adapters/sovrn/sovrn.go [hunk 1]
```diff
@@ -20,6 +20,11 @@ type SovrnAdapter struct {
 	URI string
 }
 
+type SovrnImpExt struct {
```
Comment (onkarvhanumante) [inline adapters/sovrn/sovrn.go:23]: scope of `SovrnImpExt` is within   `sovrn` package. Therefore this struct can be private

```
- type SovrnImpExt struct {
+ type sovrnImpExt struct {    //   's' is lowercase here

```

Comment (ikagotso) [reply adapters/sovrn/sovrn.go]: Nice catch, thank you. Fixed.

**Comment by onkarvhanumante** [comment]
@ikagotso  should also create docs PR to add description for `ad unit codes`

**Comment by ikagotso** [comment]
> @ikagotso should also create docs PR to add description for `ad unit codes`

Could you point me to an example of such PR? 
I can't seem to figure out where to put such information.

**Comment by onkarvhanumante** [comment]
> > @ikagotso should also create docs PR to add description for `ad unit codes`
> 
> Could you point me to an example of such PR? I can't seem to figure out where to put such information.

@ikagotso  should update bid params section  to add description for `ad unit codes`
 https://docs.prebid.org/dev-docs/bidders/sovrn.html#bid-params

This can be done by updating https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/sovrn.md. Recent PR https://github.com/prebid/prebid.github.io/pull/4725 making change to sovrn.md can be used as reference

**Comment by ikagotso** [comment]
> > > @ikagotso should also create docs PR to add description for `ad unit codes`
> > 
> > 
> > Could you point me to an example of such PR? I can't seem to figure out where to put such information.
> 
> @ikagotso should update bid params section to add description for `ad unit codes` https://docs.prebid.org/dev-docs/bidders/sovrn.html#bid-params
> 
> This can be done by updating https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/sovrn.md. Recent PR [prebid/prebid.github.io#4725](https://github.com/prebid/prebid.github.io/pull/4725) making change to sovrn.md can be used as reference

Thank you, will try to open new PR. Does this have to do anything with this PR not getting merged?

**Comment by onkarvhanumante** [comment]
> > > > @ikagotso should also create docs PR to add description for `ad unit codes`
> > > 
> > > 
> > > Could you point me to an example of such PR? I can't seem to figure out where to put such information.
> > 
> > 
> > @ikagotso should update bid params section to add description for `ad unit codes` https://docs.prebid.org/dev-docs/bidders/sovrn.html#bid-params
> > This can be done by updating https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/sovrn.md. Recent PR [prebid/prebid.github.io#4725](https://github.com/prebid/prebid.github.io/pull/4725) making change to sovrn.md can be used as reference
> 
> Thank you, will try to open new PR. Does this have to do anything with this PR not getting merged?

General acceptance criteria is to first open PR for docs change. Then we could merge PBS server related change.

**Comment by ikagotso** [comment]
> > > > > @ikagotso should also create docs PR to add description for `ad unit codes`
> > > > 
> > > > 
> > > > Could you point me to an example of such PR? I can't seem to figure out where to put such information.
> > > 
> > > 
> > > @ikagotso should update bid params section to add description for `ad unit codes` https://docs.prebid.org/dev-docs/bidders/sovrn.html#bid-params
> > > This can be done by updating https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/sovrn.md. Recent PR [prebid/prebid.github.io#4725](https://github.com/prebid/prebid.github.io/pull/4725) making change to sovrn.md can be used as reference
> > 
> > 
> > Thank you, will try to open new PR. Does this have to do anything with this PR not getting merged?
> 
> General acceptance criteria is to first open PR for docs change. Then we could merge PBS server related change.

Here it is : https://github.com/prebid/prebid.github.io/pull/4854

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**